versioned_docs/version-1.0/language/contract-updatability.md

```diff
@@ -501,3 +501,41 @@ Similar to functions, constructors are also not stored. Hence, any changes to co
 A contract may import declarations (types, functions, variables, etc.) from other programs. These imported programs are
 already validated at the time of their deployment. Hence, there is no need for validating any declaration every time
 they are imported.
+
+## The `#removedType` Pragma
+
```

Comment:
Maybe add a `:::note` here that:
- This feature is not yet finalized, and may change _after_ Cadence 1.0
- Developers should be careful when removing interfaces, as it will break code that imports this type